src/lib.cairo

```diff
@@ -2,6 +2,8 @@ mod access;
 mod account;
 mod introspection;
 mod security;
-mod tests;
 mod token;
 mod utils;
+
+#[cfg(test)]
```

Comment:
what is this line doing exactly? couldn't find it in scab docs

Reply:
Hi, it's not in our docs as it is a compiler's feature. Similarly to Rust, you can have a conditional compilation - in this case we indicate we want this modules compiled only in tests. Please take a look: https://book.cairo-lang.org/ch08-02-test-organization.html?highlight=cfg#the-tests-module-and-cfgtest